rustls/src/msgs/macros.rs

```diff
@@ -31,8 +31,8 @@ macro_rules! enum_builder {
             #[allow(dead_code)]
             $enum_vis fn as_str(&self) -> Option<&'static str> {
                 match self {
-                    $( $enum_name::$enum_var => Some(stringify!($enum_var))),*
-                    $(, $( $enum_name::$enum_var_nd => Some(stringify!($enum_var_nd))),* )?
+                    $( $enum_name::$enum_var_dbg_fmt => Some(stringify!($enum_var_dbg_fmt))),*
+                    $(, $( $enum_name::$enum_var_no_fmt => Some(stringify!($enum_var_no_fmt))),* )?
```

Comment:
As discussed in other PR #2329, I think some more test coverage is needed for these lines.

`cargo test -p rustls --all-features` continues to succeed with these lines mutated as follows:

```rs
                    $( $enum_name::$enum_var_dbg_fmt => None),*
                    $(, $( $enum_name::$enum_var_no_fmt => panic!("XXX")),* )?
```